### PR TITLE
heading update

### PR DESCRIPTION
**Pull request by divyaamunugama**

### CODE REVIEW

File: en/docs/guides/applications/service-provider-overview.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Introduction
+# Service Providers
```
Comment (nilminiwso2) [inline en/docs/guides/applications/service-provider-overview.md:1]: sentence case?

File: en/docs/guides/identity-federation/add-idp.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Introduction
+# Registering an Identity Provider
```
Comment (nilminiwso2) [inline en/docs/guides/identity-federation/add-idp.md:1]: Sentence case?

**Review by nilminiwso2 — APPROVED** [review]

File: en/docs/guides/applications/service-provider-overview.md (hunk 1 above)
Comment (divyaamunugama) [inline en/docs/guides/applications/service-provider-overview.md:1]:
```suggestion
# Service providers
```

File: en/docs/guides/identity-federation/add-idp.md (hunk 2 above)
Comment (divyaamunugama) [inline en/docs/guides/identity-federation/add-idp.md:1]:
```suggestion
# Registering an identity provider
```

**Pull request merged**